drivers/input/Kconfig.kbd_matrix

```
@@ -27,6 +27,13 @@ config INPUT_KBD_MATRIX_16_BIT_ROW
 	  Use a 16 bit type for the internal structure, allow using a matrix
 	  with up to 16 rows if the driver supports it.
 
+config INPUT_KBD_ACTUAL_KEY_MASK_DYNAMIC
+	bool "Allow runtime changes to the actual key mask"
+	help
+	  If enabled, the actual key mask data is stored in RAM, and a
+	  input_kbd_matrix_actual_key_mask_set() function is available to
+	  change the content at runtime.
```

Non-blocking - something to address with the documentation changes.
```suggestion
	  If enabled, the actual-key-mask devicetree property data is 
	  stored in RAM, and a input_kbd_matrix_actual_key_mask_set() 
	  function is available to change the content at runtime.
```

---

thanks, I'll merge this and sneak the change in the next PR